### PR TITLE
[Doc Issue][Teams App AAD Dependencies Getting Created][2635574]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/56f5e8505b4e58fd46c7f66e4613c9f7e3889c98/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 91 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/026f702a-1c9e-4b0d-926b-ac3e8c706b71) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61e6f4ec9e1602d7700bf9a82fd17edc89d783d4/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49e0d8b7-fa20-4799-8753-4a65c58d0704) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md [hunk 1]
```diff
@@ -342,7 +342,10 @@ After you've previewed the manifest file in Visual Studio, you can now sync the
 :::image type="content" source="../assets/images/Tools-and-SDK-revamp/edit-manifest-for-visual-studio/vs-update-manifest1.png" alt-text="Update manifest in teams developer portal" lightbox="../assets/images/Tools-and-SDK-revamp/edit-manifest-for-visual-studio/vs-update-manifest1.png":::
 
 > [!NOTE]
-> The changes are updated to Teams Developer Portal. If you have some manual updates in Developer Portal, that can be overwritten. In the **Warning** dialog box you can select **Overwrite and update** or **Cancel**.
+>
```
Comment (v-habibaumme) [inline msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md:345]: Add "The changes are updated to Teams Developer Portal" before to the note and the rest of the details should go in note.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/57f56aabfd80a6c611bdbc88aa505fd16ba41099/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 91 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b981d26-23f6-4c95-bac9-ba234d5bf51e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/806aaa00a0760b61ba566d9611590e5be78822ce/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 91 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b1183b3d-7c28-4628-b8f7-2e9b1c5f5025) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**